README.md

```diff
@@ -36,19 +35,37 @@ Variables that can be passed to CMake
 =========================================
 
 When running CMake it is possible to pass the following variables:
-  * `-DTARGETS=x;y;z` --- build stdlib for targets x, y and z. (Defaults to `seq;mt_pth`)
-  * `-DBUILDGENERIC=ON|OFF` --- build stdlib without using architecture specific optimisations
-  * (useful when creating distributable packages). (Default is `OFF`)
-  * `-DSAC2C_EXEC=/path/to/sac2c` --- specify `sac2c` executable directly. Otherwise CMake will
-    try to find `sac2c` on yout PATH.
-  * `-DLINKSETSIZE=n` --- set `-linksetsize n` when calling `sac2c`.  This option is responsible
-    for the number of C functions that are put in a single C file when compiling a SaC program.
-    The rule of thumb:
-    * value `0` is the fastest time-wise but potentially results in a large memory consumption
-    * value `1` reduces the memory consumption to minimum, but significantly increases compilation time.
-    
-      *Default value: 500.*
-  * `-DFULLTYPES=ON|OFF` --- add support for further types to the stdlib, such as `long` and
-    `longlong` (Default is `OFF`).
-  * `-DBUILD_EXT=ON|OFF` --- build extended stdlib (see
-    `cmake/source-core-ext.txt` for details) (Default is `ON`)
+
+* `-DTARGETS=x;y;z` --- build stdlib for targets x, y and z. (Defaults to `seq;mt_pth`)
+* `-DBUILDGENERIC=ON|OFF` --- build stdlib without using architecture specific optimisations
+   (useful when creating distributable packages). (Default is `OFF`)
+* `-DSAC2C_EXEC=/path/to/sac2c` --- specify `sac2c` executable directly. Otherwise CMake will
+  try to find `sac2c` on yout PATH.
+* `-DLINKSETSIZE=n` --- set `-linksetsize n` when calling `sac2c`.  This option is responsible
+  for the number of C functions that are put in a single C file when compiling a SaC program.
+  The rule of thumb:
+  * value `0` is the fastest time-wise but potentially results in a large memory consumption
+  * value `1` reduces the memory consumption to minimum, but significantly increases compilation time.
+  
+    *Default value: 500.*
+* `-DFULLTYPES=ON|OFF` --- add support for further types to the stdlib, such as `long` and
+  `longlong` (Default is `OFF`).
+* `-DBUILD_EXT=ON|OFF` --- build extended stdlib (see
+  `cmake/source-core-ext.txt` for details) (Default is `ON`)
+
+Continuous Integration
+=====================
+
+We make use of Github Actions for our CI pipeline, building the standard library on different systems
+anytime there is a pull request.
+
+Currently we build on the following systems: `ubuntu-18.04` and `macos-10.15`.
+
+Please look at `.github/workflows/` for more exact details on what we do.
+
+Licensing
+=========
+
+This project is open-source, contributions are welcome! Note that no warranty is given.
+
+Please look at `LICENSE.txt` for further details.
```

Comment:
Ugh, custom license... Is it really a good idea?  Wouldn't it be better to use an existing license with a similar intent?  This is a low-priority thing of course, given that the current wording satisfies github-actions.

Reply:
Yes that would be better. A quick look at https://choosealicense.com/appendix/, all of the listed licenses seem to allow for commercial use (without conditions). I'm not sure if that matter in this context, it probably doesn't matter at the moment. Do you have OS license in mind?

Reply:
Well, the license that I use for most of my projects is ISC: https://choosealicense.com/licenses/isc/ which in essence says: do whatever the hell you want with the code.  It seems to me that for a library this is pretty much the right choice.  There is no point in using stdlib commercially if the commercial use of the compiler is prohibited.  We should definitely check this with @sbscholz.  We could also decide to leave this as is, given that the current license is acceptable for GH actions.

Reply:
Yup, we need to talk to @sbscholz, marking this resolved for the moment.